content/blog/2018/03/Jenkins User Conference China Beijing Recap.adoc

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "Jenkins User Conference China Beijing Recap"
+tags:
+- event
+- juc
+author: alyssat
```

Comment:
Change author to Forest?

Reply:
it would be great if we can change Forest as author.

Reply:
You would need to create an author page for him, but after that you can just have him displayed as an author by using his user ID